app/src/main/java/com/teamagam/gimelgimel/app/map/viewModel/SendGeometryViewModel.java

```diff
@@ -76,9 +112,51 @@ public void onUndoClicked() {
   public void onSwitchChanged(boolean isChecked) {
     sLogger.userInteraction("Send geometry switch changed to " + isChecked);
     mIsSwitchChecked = isChecked;
+    notifyPropertyChanged(BR._all);
     refreshDisplayedGeometry();
   }
 
+  public void onBorderStyleSelect() {
+    try {
+      mPickBorderStyle.accept(mBorderStyle);
+    } catch (Exception ignored) {
```

Comment:
I really do not except an error here, the reason for this situation is that starting with rx2 the functional classes (Action, Consumer, Function etc.) throws an Exception by default :disappointed: